crates/sncast/src/response/errors.rs

```diff
@@ -1,5 +1,6 @@
 use crate::{ErrorData, WaitForTransactionError, handle_rpc_error};
 use anyhow::anyhow;
+use cairo_vm::Felt252;
```

Comment:
```suggestion
use starknet_types_core::felt::Felt;
```

Please replace that, `cairo_vm::Felt252` is just an alias for that anyway.